pyproject.toml

```diff
@@ -116,3 +116,4 @@ testfixtures = "*"
 tuspy = "*"
 twill = "*"
 watchdog = "*"
+fs-gcsfs = "*"
```

Comment:
That should be a conditional requirement and not appears in pyproject.toml

Reply:
How do I specify it as a conditional requirement only for tests?

Reply:
I'd maybe do it the other way round and skip the test if the dependency can't be imported.

Reply:
Yes, ideally unit tests shouldn't contact external services. We can run release tests with the conditional dependencies installed.

Reply:
BTW, `fs-gcsfs` appears to have been correctly included in dev-requirements, but not pinned requirements. Isn't that ok?

Reply:
Though we already do that for some other tests, I guess that is just a personal preference. If you want to run this one then this is ok and we do need to add it

Reply:
I've rolled back the gcsfs test in favour of an optional one.